packages/taquito-core/src/error/errors.ts

```diff
@@ -1,20 +1,10 @@
 /**
  *  @category Error
- *  @description Error that indicates an invalid user data input
- */
-export class ExternalError extends Error {
-  public name = 'ExternalError';
-  constructor(public message: string) {
-    super(message);
-  }
-}
-
-/**
- *  @category Error
- *  @description Error that indicates an invalid user data input
+ *  @description Error that indicates the data get from other servies has incorrect format
  */
-export class ValidationError extends Error {
-  public name = 'ValidationError';
+export class InternalValidationError extends Error {
+  public name = 'InternalValidationError';
+  public category = this.name;
```

Comment:
can you explain why we're assigning category the same as its name? Seems redundant

Reply:
we thought of doing this so that it is visible in the error structure that the parent is InternalValidationError
otherwise since all the parent does is use Super it would just act like `SomeError extends Error`
rather than `SomeError extends ParentError extends Error`

Reply:
couldn't you just grab super(name) from the child to and output it in the error message?

Reply:
the idea behind this was that we wouldnt need to change all of the error messages of the children that we would be extending to make visibility of the parent error

Reply:
tried super(name) and got name as Error not the parent but from grandparent class 🙈